pytorch_lightning/trainer/properties.py

```diff
@@ -483,6 +486,19 @@ def sanity_checking(self, val: bool) -> None:
     Loop properties
     """
 
+    @property
+    def fit_loop(self) -> FitLoop:
+        # TODO(@awaelchli): the current train_loop should be renamed to fit_loop
+        return self.train_loop
+
+    @property
+    def training_epoch_loop(self) -> TrainingEpochLoop:
+        return self.fit_loop.training_loop
+
+    @property
+    def training_batch_loop(self) -> TrainingBatchLoop:
+        return self.fit_loop.training_loop.batch_loop
```

Comment:
In Lightning, sometimes we use `training` (`training_step`) and others `train` (`train_dataloader`).

Do you prefer `training` over `train` for this? Do you plan to call the others `Validating*Loop` and `TestingLoop`?

Reply:
no because validating is not a noun, but training and validation both are. that to me makes the most sense. 

the training
the validation
the test